Python/Product/PythonTools/PythonTools/Environments/InterpreterView.cs

```diff
@@ -58,6 +62,28 @@ sealed class InterpreterView : DependencyObject {
             return res;
         }
 
+        private static bool FilterInterpreters(
+            InterpreterConfiguration interpreterConfiguration,
+            bool includeVirtualEnv,
+            bool includeCondaEnv,
+            bool includeIronPythonEnv
+        ) {
```

Comment:
Method name is plural, but it only processes one item. I'd go with something like `ShouldIncludeConfiguration`, or at least something not plural.

Reply:
Will be updated in next commit